src/main/java/org/opensearch/jobscheduler/JobSchedulerPlugin.java

```diff
@@ -60,10 +79,12 @@ public class JobSchedulerPlugin extends Plugin implements ExtensiblePlugin {
     private LockService lockService;
     private Map<String, ScheduledJobProvider> indexToJobProviders;
     private Set<String> indicesToListen;
+    private Map<String, JobDetails> jobDetailsHashMap;
 
     public JobSchedulerPlugin() {
         this.indicesToListen = new HashSet<>();
         this.indexToJobProviders = new HashMap<>();
+        this.jobDetailsHashMap = new HashMap<>();
```

Comment:
(Naming) Would prefer a name that tells me what the key/value relationship is.  See the above map "index" to "JobProviders".   So I think `indexToJobDetails` is better here.

Reply:
ACK